examples/misc_camera_traveling.html

```diff
@@ -0,0 +1,196 @@
+                const orthoLayer = new itowns.ColorLayer('Ortho', config);
+                view.addLayer(orthoLayer);
+            });
+
+            // Add two elevation layers.
+            // These will deform iTowns globe geometry to represent terrain elevation.
+            function addElevationLayerFromConfig(config) {
+                config.source = new itowns.WMTSSource(config.source);
+                let layer = new itowns.ElevationLayer(config.id, config);
+                view.addLayer(layer);
+            }
+            itowns.Fetcher.json('./layers/JSONLayers/WORLD_DTM.json').then(addElevationLayerFromConfig);
+            itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT_HIGHRES.json').then(addElevationLayerFromConfig);
+
+
+            // THREE.Group which stores picked points markers.
+            let pickedPointsGroup = new itowns.THREE.Group();
+            pickedPointsGroup.visible = false;
+            view.scene.add(pickedPointsGroup);
+
+            // Add a pointer at a given position.
+            function addCameraHelper() {
+                // Copy the view camera3D to modify its far plane distance.
+                // The will render a CameraHelper with shorter axes.
+                const copyCamera = view.camera.camera3D.clone();
+                copyCamera.far = 50;
+                // Creation of the new mesh :
+                const mesh = new itowns.THREE.CameraHelper(copyCamera);
+                mesh.updateWorldMatrix(true, false);
+                // Add mesh to scene :
+                pickedPointsGroup.add(mesh);
+                view.notifyChange();
+            }
+
+            // Remove all meshes from scene.
+            function clearCameraHelper() {
+                pickedPointsGroup.remove(...pickedPointsGroup.children);
+                view.notifyChange();
+            }
+
+            // Array which stores successive camera transform options (headings, tilts, ranges...
+            // see doc for CameraTransformOptions at http://www.itowns-project.org/itowns/docs/#api/Controls/CameraUtils)
+            let cameraTravel = [];
+            // cameraPositions array stores the successive camera positions.
+            // It is used to compute the distance ran at each travel step.
+            let cameraPositions = [];
+            // Camera speed in meters per second
+            let cameraSpeed = 500 / 3.6;
+            // Boolean which states if the camera is performing a travel animation or not
+            let travelOn = false;
+
+            function saveCurrentCameraTransformOptions() {
+                // Retrieve camera current transform options
+                const cameraTransformOptions = itowns.CameraUtils.getTransformCameraLookingAtTarget(
+                    view,
+                    view.camera.camera3D,
+                );
+
+                // Save camera current transform options
+                cameraTransformOptions.coord.as(view.referenceCrs, cameraTransformOptions.coord);
```

Comment:
it's necessary to convert `coord` ?

Reply:
The conversion is no longer necessary, it was a forgotten artifact from a previous version of the example.